packages/kit/src/exports/public.d.ts

```diff
@@ -275,6 +288,18 @@ export interface Emulator {
 	 * and returns an `App.Platform` object
 	 */
 	platform?(details: { config: any; prerender: PrerenderOption }): MaybePromise<App.Platform>;
+	/**
+	 * Runs before every request that would hit the SvelteKit runtime and before requests to static assets in dev mode.
+	 * In preview mode, in runs prior to all requests.
```

Comment:
The current wording is slightly confusing because the 'in dev mode' qualifier comes late. I think this rewording is a bit clearer, though it does make it sound strange that we exclude certain requests in `dev`

```suggestion
	 * During `dev`, this function runs before all requests for static assets or routes rendered by SvelteKit, but not before requests for your app's source code.
	 * During `preview`, it runs prior to all requests.
```